ios/FluentUI/Command Bar/CommandBarButtonGroupView.swift

```diff
@@ -42,12 +42,15 @@ class CommandBarButtonGroupView: UIView {
         isHidden = allViewsHidden
     }
 
+    func setEqualWidthButtons() {
```

Comment:
minor: sounds like a setter but isn't. Either update naming to make clear this is not a setter or add a parameter in some this can be set to true or false? To the latter point, are we ok that this is only one-way settable? We never need to reset it?

Reply:
Good point on the naming. I updated this and the similar method in `CommandBarCommandGroupsView` to have property with a default value that updates the views in `didSet`.